### PR TITLE
php-cs-fixer (Completing PR #881)

### PR DESCRIPTION
**Pull request by abrardev99**
Added the `php_cs_fixer` config and CI according to the template.

### CODE REVIEW

**Comment by abrardev99** [comment]
This PR forked from #881. Merging this PR should also close the #881. Thanks, @erikgaal for your contributions.

**Comment by stancl** [comment]
Did you make any changes to the workflow from the template?

**Comment by abrardev99** [comment]
> Did you make any changes to the workflow from the template?

Yes, I just added the php-cs-fixer job like https://github.com/archtechx/template/blob/master/.github/workflows/ci.yml#L42-L56.

Looks like php-cs-fixer is failing because my branch is forked from another forked PR branch. Same tests are passing in my local forked repo PR https://github.com/abrardev99/tenancy/pull/3. Maybe we should merge it as is. And I believe the php-cs-fixer check will pass in #881.

**Comment by stancl** [comment]
> Yes, I just added the php-cs-fixer job like

It seems different — I was asking **what changes you made**

**Comment by stancl** [comment]
I mean changes compared to the template. For example, the email is different. So I'm asking why, and what other changes you made.

**Comment by abrardev99** [comment]
> I mean changes compared to the template. For example, the email is different. So I'm asking why, and what other changes you made.

I just changed the email. The rest of the workflow is the same as in the template.

**Comment by stancl** [comment]
Make it the same as it is in the template.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@ name: CI
 
 env:
   COMPOSE_INTERACTIVE_NO_CLI: 1
+  PHP_CS_FIXER_IGNORE_ENV: 1
```
Comment (stancl) [inline .github/workflows/ci.yml:5]: I think this can be removed now

Comment (stancl) [reply .github/workflows/ci.yml:5]: Try it in your branch and then we can update the template and this repo

Comment (abrardev99) [reply .github/workflows/ci.yml:5]: Yes, checks are passing after removing this. I'll update the template.

Comment (stancl) [reply .github/workflows/ci.yml:5]: Both this repo and the template

Comment (abrardev99) [reply .github/workflows/ci.yml:5]: Got it.

Comment (abrardev99) [reply .github/workflows/ci.yml:5]: PRs sent to both repos. 

https://github.com/archtechx/tenancy/pull/904
https://github.com/archtechx/template/pull/4

**Pull request merged**